wordpress-mobile/WordPress-iOS
# Domain Registration: Display server errors

Related to #10536

WordPressKit-iOS PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/261

To test:
There is a "Register Domain" button appearing for premium accounts who have free premium domain or other accounts with domain credits.
1. Open domain registration suggestions
1. Search a random domain name and select it
1. In domain registration form's fields enter invalid characters like `!@#$%^&*`
1. Submit the form
1. Check if you see errors like: `This field can only contain the following characters: “a-z A-Z 0-9 . , ‘ – [space]`

Before | After
-- | -- 
<img width="300" src="https://user-images.githubusercontent.com/1845482/87795530-7dae5900-c848-11ea-91a0-3b675b62904b.png">|<img width="300" src="https://user-images.githubusercontent.com/1845482/87795526-7b4bff00-c848-11ea-926f-89d289dcb4c3.png">



PR submission checklist:
